### DISCUSSION TITLE
group moving when i use cannon

### DISCUSSION THREAD

**Discussion by neciszhang** [pmndrs/use-cannon#241] [Q&A] [answered]
Hi, I export a gltf model, like this:
`
 <group>
     <meshA ref={refA}/>
     <meshB ref={refB}/>
 </group>
`
when i move the group ,set the position (0,2,0), meshA and meshB move right! But meshA's physics and meshB's physics don't move,it is still in it's original position (0,0,0).
I only want to move group,because group contain many meshes.
Could you give me some ideas?Thanks a lot!

**Comment by stockhuman** [comment]
Are you using a compound body?

**Reply by neciszhang** [reply]
Thank you! Drcmda explained the result I wanted.

**Comment by drcmda** [comment, marked as answer]
cannon assumes that objects are absolute, they can't be relatively nested. the object structure of physical main objects should be flat. the individual objects can be nested. the reason for this is that cannon is controlling these objects, and it cannot be aware of all the layers around it. you give it an object and its now under the full control of the physics provider.

**Reply by neciszhang** [reply]
I understand. Thank you very much! I
